src/components/tabs/tabs.ts

```diff
@@ -643,6 +643,8 @@ export class Tabs extends Ion implements AfterViewInit, RootNode, ITabs, Navigat
    * @private
    */
   _getSelectedTabIndex(secondaryId: string, fallbackIndex: number = 0): number {
+    secondaryId = secondaryId || '';
```

Comment:
Any reason you didn't use default value for parameter? Like `secondaryId: string = '', fallbackIndex: number = 0`